portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,13 +40,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String maxContent = request.getParameter("max");
```

Comment:
I think Alfred may have commented this in someone else's PR but it would be great to move these field names, such as "max" and "content" to a class level static final string.